platform-designs/user-flow/README.md

```diff
@@ -0,0 +1,9 @@
+# User Flow 
+
+To illustrate how the platform will work and how a user would navigate through it, we have created a user flow. 
+
+This can be seen below:
+
+![](../platform-designs/user-flow/user-flow.png)
```

Comment:
This looks the right formatting but I'm not 100% sure the image will render. Where did you store it?

Reply:
@GeorgiaHCA I uploaded in the user flow folder...